src/api/libcellml/importer.h

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <string>
+
+#include "libcellml/logger.h"
+#include "libcellml/types.h"
+
+namespace libcellml {
+
+/**
+ * @brief The Importer class.
+ *
+ * The Importer class is for representing a CellML Importer.
+ */
+class LIBCELLML_EXPORT Importer: public Logger
+{
+public:
+    ~Importer() override; /**< Destructor */
+    Importer(const Importer &rhs) = delete; /**< Copy constructor */
+    Importer(Importer &&rhs) noexcept = delete; /**< Move constructor */
+    Importer &operator=(Importer rhs) = delete; /**< Assignment operator */
+
+    /**
+     * @brief Create a @c Importer object.
+     *
+     * Factory method to create a @c Importer.  Create a
```

Comment:
"... a[n] @c Importer"?

Same everywhere where "Importer" is mentioned (e.g. for the return description).

Reply:
671341a6